docs/web.rst

```diff
@@ -737,28 +742,49 @@ that :meth:`Request.post` reads the whole payload in memory, resulting in possib
 should use :meth:`Request.multipart` which returns a :ref:`multipart reader
 <aiohttp-multipart>`::
 
-    async def store_mp3_handler(request):
-
-        reader = await request.multipart()
-
-        # /!\ Don't forget to validate your inputs /!\
+    async def store_multipart_handler(request):
+        """Handle a multipart form."""
+        if request.method == 'POST':
```

Comment:
I think this is a bad pattern to recommend. You should have this condition on router level.